### PR TITLE
feature/864_repo_refactoring_multi_agent___install_from_sources

### PR DESCRIPTION
**Pull request by frbattid**
* Partially implements issue #864 
* No tests are required
* Assignee @pcoello25 but LGTM from @vgarciag is also required

### CODE REVIEW

**Comment by pcoello25** [comment]
LGTM

File: doc/cygnus-ngsi/installation_and_administration_guide/install_from_sources.md [hunk 1]
```diff
@@ -134,6 +135,13 @@ In both cases, the installation is completed by copying the `cygnus-flume-ng` sc
 
 [Top](#top)
 
+###<a name="section4.4"></a>Known issues
+It may happen while compiling either `cygnus-common` either `cygnus-ngsi` the Maven JVM has not enough memory. This can be chaged as detailed at the [Maven official documentation](https://cwiki.apache.org/confluence/display/MAVEN/OutOfMemoryError):
```
Comment (pcoello25) [inline doc/cygnus-ngsi/installation_and_administration_guide/install_from_sources.md]: chaged -> changed?

Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/install_from_sources.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/955/commits/626cdcc1c89100bd68ad03149cd7cd4223add6e9

**Comment by vgarciag** [comment]
LGTM

**Pull request merged**